random.go

```diff
@@ -0,0 +1,17 @@
+package gocommon
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+// GenRandNumber can generate a random number with a positive boundary value.
```

Comment:
```suggestion
// GenRandNumber generates a random positive number less than the boundary value.
```
I guess we can remove "can", the rest lgtm.